fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/systems.cfg

```diff
@@ -517,6 +517,7 @@ stick_shaker = 1
 
 [DEICE_SYSTEM]
 structural_deice_type = 1 ; 0 = None, 1 = Heated Leading Edge, 2 = Bleed Air Boots, 3 = Eng Pump Boots
+structural_deice_rate = 0.01666 ; Pct reduction of ice per second -- Clears airframe of ice in 60 seconds
```

Comment:
Does anyone have an opinion on this? The MSFS default is 0.00128 %/sec, which seems incredibly slow. Of course I don't think this has a real world equivalent - as any deice would be done by external vehicles rather than the anti-ice systems.